src/Meteor.js

```diff
@@ -109,29 +109,37 @@ const Meteor = {
       ...options,
     });
 
-    try {
-      const NetInfo = require('@react-native-community/netinfo').default;
-
-      if (options.reachabilityUrl) {
-        NetInfo.configure({
-          reachabilityUrl: options.reachabilityUrl,
-          useNativeReachability: true,
-        });
-      }
+    if (options.NetInfo !== null) {
+      try {
+        const NetInfo = getNetInfo();
```

Comment:
Should you pass the `options.NetInfo` in here, for using a custom NetInfo?

Reply:
This works as the following:

if `options.NetInfo` is `null`, then `NetInfo` is not used at all, even not required, fixing #143 
if `options.NetInfo` is `undefined` then the default `NetInfo` is imported and configured
if `options.NetInfo` exists then it will be used. This is used to inject an own NetInfo instance with custom config

The `getNetInfo()` is only to decice between option 2 or 3 and wrap the require in a function to prevent bundling related erorrs that could prevent #143 to be fixed

Reply:
Ok, I do not understand how `getNetInfo()` can return the object `options.NetInfo` like this... Should we do `getNetInfo(options.NetInfo)` here or take the `options.NetInfo` into account inside the definition of `getNetInfo`?

Reply:
Now I get it :see_no_evil: You're right, it should be `getNetInfo(options.NetInfo)`